lib/protocol/events.js

```diff
@@ -420,6 +420,18 @@ function eventsProtocolFactory (
         return self.publishGraphEvent(graphId, 'progress.updated', data.nodeId, data);
     };
 
+    EventsProtocol.prototype.subscribeSelEvent = function (pollerId, nodeId, callback) {
+        assert.isMongoId(pollerId, 'subscribeSelEvent requires pollerId');
+        assert.isMongoId(nodeId, 'subscribeSelEvents requires nodeId');
+        assert.func(callback, 'subscribeSelEvents requires callback function');
+
+        return messenger.subscribe(
+            Constants.Protocol.Exchanges.Events.Name,
+            'polleralert.sel.updated.critical.' + pollerId + '.' + nodeId,
```

Comment:
Is it possible to make severity ("critical" here) also a parameter passed by callers? That might be more flexible for different user scenarios.